app/models/filters/health_touch_point_exports_filter.rb

```diff
@@ -10,7 +10,7 @@ class HealthTouchPointExportsFilter < DateRange
     attribute :search_scope
 
     def touch_points_for_user user
-      raise 'Search Scope required' unless search_scope
+      return search_scope.none unless search_scope.present?
```

Comment:
Not sure it matters, but seems odd to return a scope in this case, and an array otherwise?

Reply:
Yep, I got myself confused.  Reverting both to `[]`